CONTRIBUTING.md

```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
 
 https://github.com/skydive-project/skydive/issues
 
+Making Changes:
+---------------
+
+Make sure that the tests and the linters pass:
+
+make fmt
+make test
+make test.functionals
+
+Make your changes, then ensure make test still pass. If you're satisfied with your changes, push them to your fork.
```

Comment:
I would say:

Once ready push your changes to your Skydive fork our CI will check them.